import { TestThrowError } from "@lib/err";

/**
 * Error handling test cases
 */
export function Error(value: string) {
  TestThrowError(value);
}
